python/servo/bootstrap_commands.py

```diff
@@ -63,11 +63,13 @@ def download(desc, src, writer, start_byte=0):
 
         if not dumb:
             print()
-    except urllib2.URLError:
-        print("Error downloading Rust compiler; are you connected to the internet?")
-        sys.exit(1)
     except urllib2.HTTPError, e:
         print("Download failed (%d): %s - %s" % (e.code, e.reason, src))
+        if e.code == 403:
+            print("Platform support information: https://doc.rust-lang.org/book/getting-started.html#platform-support")
```

Comment:
I think "No Rust compiler binary available for this platform. Please see https://github.com/servo/servo/#prerequisites" would be more useful.